0000-report-inboxes.md

```diff
@@ -0,0 +1,72 @@
+* Mods might take local action on a report and mark it as resolved even in cases where a user should be banned from the entire instance. In this case, admins are very unlikely to see the report.
+
+# Guide-level explanation
+
+### When creating reports, users will be able to select if it's a mod report, or an admin report (or both)
+
+![image](https://github.com/sunaurus/lemmy-rfcs/assets/5356547/9a21b527-6c88-4024-b287-3371d77688f4)
+
+### Instead of the current single report inbox, there will be three different kinds of inboxes
+
+* Admin reports - show all reports sent to admins (only visible to admins)
+* Mod reports - show all reports sent to mods for any communities the user moderates
+* All reports - Shows a read-only view of all (admin and mod) reports, only visible to admins
+   * This is a read-only version of the current 0.19.3 admin report view, and would allow admins to still keep an eye on mod actions on their instance if they wish
+    
+The UI wouldn't need to change for mods, but for admins, there would be a new selection at the top of the reports page (the "mod reports" tab would only be visible if the admin is also a mod in any community):
+![image](https://github.com/sunaurus/lemmy-rfcs/assets/5356547/cc4ad68c-6e85-4cd9-b324-131c06951cb3)
+
+### Resolving reports should be more granular
+
+* Reports in the "admin reports" tab can only be resolved for admins of the local instance
+   * To reduce overhead, **banning the reported user on the user's home instance + removing reported content should automatically resolve reports for remote admins as well**
+* Reports in the "mod reports" tab can only be resolved by relevant mods. Admins can only resolve these if they are also explicit mods in the relevant communities.
+   * To reduce overhead, **admins banning the reported user on the community instance OR the user's home instance + removing reported content should automatically resolve reports for mods as well**
+* Reports in the "all reports" tab can not be resolved, they are only there for a read-only overview
+   * This is to prevent cases of admins accidentally preventing mods from moderating according to their own community rules
+   * Admins can still always explicitly take over communities by making themselves mods, in this way, they are able to handle mod reports for any abandoned communities, etc
+
+### Mods should be able to escalate reports to admins
+
+This would generate a corresponding report in the admin inbox.
+
+# Reference-level explanation
+
+* In the UI, changes are needed for both reporting as well as the reports inbox views
+* In the database and API, we should split reports by intended audience
+* Federation needs to be changed as well in order to allow distinguishing the report target audience
+
+# Drawbacks
+
+It might make reporting slightly more confusing for end users - the mod/admin distinction might not be fully clear to all.
```

Comment:
Another idea:

What if instead of saying 
1. Report to admins
2. Report to mods

We say:
1. Does this break a rule in this community? (this will report to mods)
2. Does this break a rule on this server? (this will report to admins)

If we made rules actual data instead of just text, you could even imagine a follow-up question lik

1. Does this break a rule in this community?
    1. Breaks Rule 1: "Only posts relevant stuff".
    2. Breaks Rule 2: "Something something rule 2"
3. Does this break a rule on this server? (this will report to admins)
    1. Breaks Rule 1: "No bigotry"
    2. Breaks Rule 2: "No illegal activity"

Just as an example, hope that makes sense. However that could perhaps be pushed to further work and not necessarily be part of this RFC.

Reply:
I like the simple, less verbose options better: `Report to Both`, `Report to Admins only`, `Report to Community Mods only`

Reply:
I think the problem is that it's a non-obvious choice that the user might not be truly capable of answering. I imagine most users would ask themselves "how do I know whether it should go to an admin or a mod?". Some users might not even know the difference between an admin and a mod.

However, the user is more likely to be able to answer or at least find the answer when asked about the rules of the community or the rules of the server.

The follow-up questions for each specific rule would also make it a lot easier for the user to answer, as they would have pre-defined options. The user wouldn't even have to go look for the rules.

It's not like this is rocket science, Reddit does literally the same thing. I don't think we need to reinvent the wheel here.

So yes, it is more verbose but also a lot more user-friendly. Generally Lemmy is not too user-friendly at the moment and I think it can do a lot better in this aspect.